cinema/serializers.py

```diff
@@ -135,3 +195,35 @@ def create(self, validated_data):
 
 class OrderListSerializer(OrderSerializer):
     tickets = TicketListSerializer(many=True, read_only=True)
+
+
+class AuthTokenSerializer(serializers.Serializer):
+    email = serializers.EmailField(label=_("Email"), write_only=True)
+    password = serializers.CharField(
+        label=_("Password"), style={
+            "input_type": "password"
+        },
+        trim_whitespace=False, write_only=True
+    )
+    token = serializers.CharField(label=_("Token"), read_only=True)
+
+    def validate(self, attrs):
+        email = attrs.get("email")
+        password = attrs.get("password")
+
+        if email and password:
+            user = authenticate(
+                request=self.context.get("request"),
+                username=email,
```

Comment:
The `authenticate` function is incorrectly using 'username=email'. It should be 'email=email' to match the field used for authentication.